consent-ui/src/app/ais/sca-result-page/sca-result-page.component.ts

```diff
@@ -1,53 +1,56 @@
-import {Component, OnInit} from '@angular/core';
-import {FormBuilder, FormGroup, Validators} from "@angular/forms";
-import {ActivatedRoute} from "@angular/router";
-import {StubUtil} from "../common/stub-util";
-import {ConsentAuthorizationService} from "../../api";
-import {ApiHeaders} from "../../api/api.headers";
-import {SessionService} from "../../common/session.service";
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+import { StubUtil } from '../common/stub-util';
+import { ConsentAuthorizationService } from '../../api';
+import { ApiHeaders } from '../../api/api.headers';
+import { SessionService } from '../../common/session.service';
 
 @Component({
   selector: 'consent-app-sca-result-page',
   templateUrl: './sca-result-page.component.html',
   styleUrls: ['./sca-result-page.component.scss']
 })
-
-export class ReportScaResultComponent implements OnInit{
+export class ReportScaResultComponent implements OnInit {
   public static ROUTE = 'sca-result';
 
   reportScaResultForm: FormGroup;
   private authorizationSessionId: string;
 
-  constructor(private sessionService: SessionService,
-              private consentAuthorizationService: ConsentAuthorizationService,
-              private activatedRoute: ActivatedRoute,
-              private formBuilder: FormBuilder) {
-  }
+  constructor(
+    private sessionService: SessionService,
+    private consentAuthorizationService: ConsentAuthorizationService,
+    private activatedRoute: ActivatedRoute,
+    private formBuilder: FormBuilder
+  ) {}
 
   ngOnInit() {
     this.reportScaResultForm = this.formBuilder.group({
       tan: ['', Validators.required]
     });
 
-    this.authorizationSessionId = this.activatedRoute.parent.snapshot.paramMap.get("authId");
+    this.authorizationSessionId = this.activatedRoute.parent.snapshot.paramMap.get('authId');
   }
 
   submit(): void {
-      this.consentAuthorizationService.embeddedUsingPOST(
+    this.consentAuthorizationService
+      .embeddedUsingPOST(
         this.authorizationSessionId,
         StubUtil.X_REQUEST_ID, // TODO: real values instead of stubs
         StubUtil.X_XSRF_TOKEN, // TODO: real values instead of stubs
         this.sessionService.getRedirectCode(this.authorizationSessionId),
```

Comment:
Maybe would be better to have a separate var for redirect code, since it makes the code more readable